apps/native-component-list/src/components/FunctionConfigurator.tsx

```diff
@@ -0,0 +1,212 @@
+import Checkbox from 'expo-checkbox';
+import React, { useCallback } from 'react';
+import { StyleSheet, Text, View, TouchableOpacity, Platform } from 'react-native';
+
+import Colors from '../constants/Colors';
+import {
+  EnumParameter,
+  FunctionArgument,
+  FunctionParameter,
+  NumberParameter,
+  Parameter,
+  PrimitiveArgument,
+  PrimitiveParameter,
+  StringParameter,
+} from './FunctionDemo.types';
+
+type FunctionConfiguratorProps = {
+  parameters: FunctionParameter[];
+  value: FunctionArgument[];
+  onChange: (name: string | [string, string], newValue: PrimitiveArgument) => void;
+};
+
+export default function FunctionConfigurator({
```

Comment:
The idea behind `Configurator` was to do simple thing: return an object with options based on given config, without knowing what to do with the object and how it's going to be used, so I think the `Function` prefix is not necessary here.

Reply:
I've modified the `Configurator` and at this point it operates on these fixed types (that are very sophisticated at the first glimpse) that are directly connected with the `FunctionDemo`. I've changed the name to the `Configurator`, but I wonder what you think now 🤔